### ISSUE TITLE
About the model accuracy problem

### ISSUE THREAD

**Issue by 1079174401** [Om-Prakash08/Electricity-fraud-detection-using-CNN#2]
Why do ANN and 1-DCNN perform well on the training set in terms of loss value and accuracy, but become much worse on the test set?
